src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -377,9 +377,9 @@
                       :height 40}}]
            [communities.icon/community-icon community])]
         [rn/view {:padding-right 14 :flex 1}
-         [rn/text {:style {:font-weight "700" :font-size 17}}
+         [rn/text {:style {:font-weight "700" :font-size 17 :color quo.colors/black}}
           name]
-         [rn/text description]]]
+         [rn/text {:style {:color quo.colors/black}} description]]]
```

Comment:
Should this style live in `status-im.ui.screens.chat.styles.message.message`. ?
Should we add case for dark mode (By using `quo.colors/custom-color`). ?